### ISSUE TITLE
Emma last things

### ISSUE THREAD

**Issue by cgruppioni** [harvard-lil/h2o#140]

**Comment by cgruppioni** [comment]
Deleting a casebook feature

**Comment by cgruppioni** [comment]
Data re-mirations

**Comment by bjohnson747** [comment]
Add a TOC Word style to the TOC in the DOCX export

**Comment by bjohnson747** [comment]
Highlight text overlap: https://github.com/harvard-lil/h2o/issues/118

**Comment by bjohnson747** [comment]
Footnote operation: https://github.com/harvard-lil/h2o/issues/117

**Comment by emmalemma** [comment]
Apologies for the delay on this stuff— I think it's all taken care of, but as always let me know if you have questions.

- **I just filed PR #144 which contains fixes for #118, #117 and Casebook deletion.**

- **For TOC styling,** the normal Word styles (TOC Heading, TOC 1, TOC 2, etc.) seem to be correctly applied— their names aren't visible when the TOC text is highlighted, but editing them in the styles pane of `default.docx` appears to have the intended effect of styling the TOC in exports.

- **I did a final data migration** from the 100 casebooks dump last week, which included e.g. casebook owner names. The process is pretty straightforward, here's what I did:

I loaded the legacy casebook dump from scratch on my machine:
```
bin/rails db:purge
psql -d h2o_dev < h2o_brett-2017071702.sql
```
I migrated playlists locally:
```
bin/rails h2o:migrate_playlists
```
(This takes quite a while as it converts the annotations. It also printed many warnings of invalid annotations— AFAICT these are in fact invalid annotations that are present in the dump and which don't appear on the old site.)

At this point the migration is done on my local machine. I then dumped the migrated data and uploaded it to the `lil.tools` instance:
```
pg_dump h2o_dev -t users -t content_nodes -t content_annotations -t content_collaborators -t cases -t text_blocks -t defaults > import_dump.sql
gzip import_dump.sql
scp import_dump.sql.gz emma@50.16.134.118:~/
```
And then imported the migrated dump:
```
RAILS_ENV=production /usr/local/rbenv/shims/bundle exec bin/rails db < ~/import_dump.sql
```

As I mentioned, in a context where a clean copy of the old data already exists in the target database, most of these steps are unnecessary— merely running `h2o:migrate_playlists` will populate the new tables in-place.
